spec/features/reply_from_councillor_spec.rb

```diff
@@ -14,9 +15,8 @@
     end
   end
 
-  background { comment.confirm! }
-
   scenario "it's entered by an admin and displayed on the application page" do
+    comment.confirm!
```

Comment:
this was in a background block but only needed to be run for the first test - hence why it's here.

The 2nd test i.e. `it’s loaded from WriteIt by an admin` is already started with a confirmed comment.